libibverbs/verbs.h

```diff
@@ -84,6 +85,54 @@ struct ibv_gid_entry {
 	uint32_t ndev_ifindex;
 };
 
+enum {
+	VERBS_LOG_LEVEL_NONE,
+	VERBS_LOG_ERR,
+	VERBS_LOG_WARN,
+	VERBS_LOG_INFO,
+	VERBS_LOG_DEBUG,
+};
+
+#define verbs_log(ctx, level, format, arg...)                                         \
```

Comment:
Don't we need to add __printf(3, 4) attribute to silence some GCC 10 warnings?

Reply:
I guess we would if it was a function instead of a macro.